### PR TITLE
FEATURE: Rake themes installer

### PR DESCRIPTION
**Pull request by rrooding**
This makes it possible to install themes and theme components from the command line. It accepts a YAML file with the themes you want to install.

```
$ bundle exec rake themes:install < config/themes.yml
  example1: is already installed
  example2: installed from https://github.com/example/theme.git
  doesntexist: Error cloning git repository, access is denied or repository is not found

Results:
 Installed: 1
 Updated:   0
 Skipped:   1
 Errors:    1
```

### CODE REVIEW

**Comment by rrooding** [comment]
The failures don't seem related to the PR but I can't restart the tests in Travis.

The failing tests are both Qunit tests for something this PR doesn't touch

File: spec/models/theme_spec.rb [hunk 1]
```diff
@@ -59,6 +59,8 @@
 
   end
 
+  it { is_expected.to belong_to(:remote_theme).dependent(:destroy) }
```
Comment (eviltrout) [inline spec/models/theme_spec.rb]: Generally we find such tests to be of little value (since they are 1:1 with code) and do not include them.

Comment (rrooding) [reply spec/models/theme_spec.rb]: Removed 👍

**Comment by eviltrout** [comment]
Thanks, merged!

**Pull request merged**